capstone-backend/pom.xml

```diff
@@ -57,11 +57,6 @@
       <artifactId>slf4j-simple</artifactId>
       <version>1.7.25</version>
     </dependency>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <version>29.0-jre</version>
-    </dependency>
```

Comment:
Confirmed with Steven to figure out that the guava dependency was not causing the build to fail and Guava takes the project quite a long time to deploy locally or remote.